packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts

```diff
@@ -83,6 +105,7 @@ export async function handleAntiRaidNuke(
 				lng: locale,
 			})}\n\n${prefixedParameterStrings.join('\n')}`,
 		});
+		return;
 	}
```

Comment:
Can't mark beyond this point, but can't we just throw here instead?

Reply:
Fixed in 67ee213